testsuite/features/step_definitions/command_steps.rb

```diff
@@ -603,11 +603,69 @@
   raise "Not found:\n#{output}" unless output.include?(text)
 end
 
+Then(/^tftp should be running on the proxy$/) do
+  node = get_target('proxy')
+  case $product
+  # TODO: Should we handle this in Sumaform?
+  when 'Uyuni'
+    step %(I enable repositories before installing branch server)
+    cmd = 'zypper --non-interactive --ignore-unknown remove atftp && ' \
+          'zypper --non-interactive install tftp && ' \
```

Comment:
While you are at using steps, I think we have a step for installing packages...

Reply:
Here is important to use `--ignore-unknown`.  Let's keep it like this by now.

Reply:
you could add `--ignore-unknown` to our package removal step...

Reply:
That could hide a problem in other contexts.
I don't like to extrapolate from the particular to the general;
I have found that I need this option, let's keep it here well localized in this specific context.